Makefile

```diff
@@ -1,4 +1,4 @@
-.PHONY: all submodules server client clean purge test help default fb
+.PHONY: all server client prepare build clean purge help default
```

Comment:
I see `purge` is gone, and `prepare` and `build` are not there... yet :) 
Maybe they exist in your local version, just need to push upstream?